components/fctl/cmd/stack/users/upsert.go

```diff
@@ -29,10 +29,11 @@ func NewUpsertController() *UpsertController {
 }
 
 func NewUpsertCommand() *cobra.Command {
```

Comment:
Why `upsert` ?

Reply:
Because the user could not exist in the organization.

Do we want to enforced the invitation process ?